src/PKSim.Core/Model/IndividualParameterSameFormulaOrValueForAllSpecies.cs

```diff
@@ -5,8 +5,5 @@ public class IndividualParameterSameFormulaOrValueForAllSpecies
       public int ContainerId { get; set; }
       public string ContainerPath { get; set; }
       public string ParameterName { get; set; }
-      public bool IsSameFormula { get; set; }
```

Comment:
@Yuri05 I never care if this is same formula or sane value. I care if it's the same or not (I think)

Reply:
ok